pkg/create/templates/test.go

```diff
@@ -106,7 +114,9 @@ test_connection() {
   local result=1
   while [ $attempt -le $max_attempts ]; do
     echo "Sending GET request to http://$(container_ip):${test_port}/"
-    response_code=$(curl -s -w %{http_code} -o /dev/null http://$(container_ip):${test_port}/)
+    # here, we run the curl inside a container for people running on boot2docker (who can't access the container's IP directly)
+    #response_code=$(curl -s -w %{http_code} -o /dev/null http://$(container_ip):${test_port}/)
+    response_code=$(docker run --rm -it sequenceiq/alpine-curl curl -s -w %{http_code} -o /dev/null http://$(container_ip):${test_port}/)
```

Comment:
I understand (even though it's just a template, so people are free to change it before running it).
what about reverting to the original behaviour, and just displaying a warning to osx users suggesting them to run the curl inside a container if they can't access the container's ip directly ?